easybuild/tools/options.py

```diff
@@ -265,6 +268,9 @@ def basic_options(self):
                       'pathlist', 'store_or_None', [], 'r', {'metavar': 'PATH[%sPATH]' % os.pathsep}),
             'robot-paths': ("Additional paths to consider by robot for easyconfigs (--robot paths get priority)",
                             'pathlist', 'add_flex', self.default_robot_paths, {'metavar': 'PATH[%sPATH]' % os.pathsep}),
+            'search-paths': ("Additional paths to consider in search for easyconfigs (--robot and --robot-paths paths "
+                             "get priority)",
```

Comment:
`Additional locations to consider in --search (next to --robot and --robot-paths paths)`